test/ComposerGpgVerifyTest/VerifyTest.php

```diff
@@ -0,0 +1,445 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ComposerGpgVerifyTest;
+
+use Composer\Composer;
+use Composer\Config;
+use Composer\Script\Event;
+use Composer\Script\ScriptEvents;
+use ComposerGpgVerify\Verify;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Process\Process;
+
+/**
+ * @covers \ComposerGpgVerify\Verify
+ */
+final class VerifyTest extends TestCase
```

Comment:
That is the current `HEAD`, since composer puts the `HEAD` to the installation version. I can obviously extract the git version from the lockfile too, but that doesn't guarantee that what is checked out is valid